ot3_state_manager/ot3_state_manager/messages.py

```diff
@@ -102,6 +117,54 @@ def to_bytes(self) -> bytes:
         return struct.pack(">BHB", MessageID.SYNC_PIN.message_id, 0, self.state)
 
 
+@dataclass
+class GetAxisLocationMessage(Message):
+    """Message to get current location of axis."""
+
+    axis: OT3Axis
+
+    def __eq__(self, other: Any) -> bool:
```

Comment:
do we really need this? dataclasses should generate this for free. same with hash.

Reply:
e540a74f5db3251c4db29cbc66ae381c83d2bcba